docs/markdown/Pkgconfig-module.md

```diff
@@ -58,6 +58,7 @@ keyword arguments.
   generating the uninstalled pkg-config file. Since *0.54.0*
 - `dataonly` field. (*since 0.54.0*) this is used for architecture-independent
    pkg-config files in projects which also have architecture-dependent outputs.
+- `conflicts` (*since 0.36.0*) list of strings to be put in the `Conflicts` field.
```

Comment:
We should also note that prior to 0.54, Meson was incorrectly emitting a warning when this kwarg was being used.

Reply:
Added. Force push.